holidayextras/holidayextras.github.com
# Update Jekyll & more

#### What does this PR do? (please provide any background)
Well! GitHub pages has it's own gem that helps you update it easier. The latest version of jekyll has a shedload of features too, including sass support.

What's change:
- Changed to using `github-pages` gem
- LESS converted to SASS ( built in to jekyll )
- Changed code hightlighter from js to using Rouge ( built in to jekyll )
- Improved build scripts
- General cleanup

#### What tests does this PR have?
N/a

#### How can this be tested?
- Remove node modules `rm -rf node_modules/`
- Optional remove jekyll `gem uninstall jekyll` and select all versions
- Install all using single script `npm i`
- Run locally `npm start`

Hopefully very little should look different. The only UI change is the syntax highlighter now uses rouge.

#### Screenshots / Screencast


#### What gif best describes how you feel about this work?
![](http://45.media.tumblr.com/tumblr_l670y2De1q1qc39fro1_500.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
